metabase/metabase
# Add Type Filter to Search

### Description

Adds a `Content Type` filter dropdown to replace the `SearchApp`'s current filter sidebar.

### How to verify

* Type something into the search and click `Enter`.
* You should see the `Content type` filter on the right side, with which you should be able to filter the search results based on the model type. 
* The URL should also change when filters are applied, meaning you should be able to refresh the page and see the same search after.

### Demo

https://github.com/metabase/metabase/assets/25306947/71ad91b6-3e67-4555-8ff5-b7ff203178a4




### Checklist
